data-products/overview.md

```diff
@@ -1,5 +1,39 @@
 # Overview
 
+This page summarizes the data products available from the NISAR mission.
+
+## Level 3
+
+Level 3 products represent geophysical parameters derived from radar-specific parameters and geocoded to a geographic grid.
+
+- The [Soil Moisture (SME2)](level-3/sme2.md) product contains soil moisture estimates derived from NISAR data using three different algorithms at agricultural field-scale (200m) posting. Soil moisture information can be used for reservoir management, early warning of droughts, irrigation scheduling, and crop yield forecasting.
+
+## Level 2
+
+Level 2 products are derived radar-specific parameters resampled and geocoded to a geographic grid. They can be directly overlaid on a map and combined with other data in geographic coordinates.
+
+- The [Geocoded SLC (GSLC)](level-2/gslc.md) product represents the single look complex signal return for each polarization in geographic coordinates. The GSLC product enables users to perform backscatter amplitude analysis, and can be combined with other similar GSLC products to derive interferograms and change maps.
```

Comment:
Since it's spelled out in RSLC I think it should be spelled out here, too. 
```suggestion
- The [Geocoded Single Look Complex (GSLC)](level-2/gslc.md) product represents the single look complex signal return for each polarization in geographic coordinates. The GSLC product enables users to perform backscatter amplitude analysis, and can be combined with other similar GSLC products to derive interferograms and change maps.
```